membership-attribute-service/app/services/SupporterProductDataService.scala

```diff
@@ -17,11 +17,9 @@ import scala.concurrent.ExecutionContext
 class SupporterProductDataService(client: AmazonDynamoDBAsync, table: String, mapper: SupporterRatePlanToAttributesMapper)
   (implicit executionContext: ExecutionContext) {
 
-  implicit val jodaNumberFormat: DynamoFormat[LocalDate] =
-    DynamoFormat.coercedXmap[LocalDate, Long, IllegalArgumentException](epochSeconds => Instant.ofEpochSecond(epochSeconds).toDateTime.toLocalDate)(
-      // We need epoch time in seconds https://docs.aws.amazon.com/amazondynamodb/latest/developerguide/time-to-live-ttl-before-you-start.html
-      _.toDateTimeAtStartOfDay.getMillis / 1000
-    )
+  implicit val jodaStringFormat: AnyRef with DynamoFormat[LocalDate] = DynamoFormat.coercedXmap[LocalDate, String, IllegalArgumentException](LocalDate.parse)(
```

Comment:
why AnyRef?

Reply:
Not sure I think Intellij added that and I didn't notice. I've removed it now.